features/attached_commands.feature

```diff
@@ -0,0 +1,85 @@
+@uses.config.contract_token
+Feature: Command behaviour when attached to an UA subscription
+
+    Scenario: Attached refresh in a trusty lxd container
+        Given a trusty lxd container with ubuntu-advantage-tools installed
+        When I attach contract_token with sudo
+        And I run `ua refresh` as non-root
+        Then I will see the following on stderr:
+            """
+            This command must be run as root (try using sudo)
+            """
+            #When The machine is attached
```

Comment:
Chad, there is a step with that `Scenario: Attached detach in a trusty lxd container` ...  
`And I run `ua detach` as non-root`

Reply:
I agree that we could have some scenarios joined together, but we could also revisit/refactor it in the future (maybe when we are going to have VMs) because currently we just have tests running in container(s).